blog/2022-07-08-entity_naming.md

```diff
@@ -0,0 +1,61 @@
+---
+author: Erik Montnemery
+authorURL: https://github.com/emontnemery
+title: "Number entity refactoring to support unit conversion"
+---
+
+## New recommendations for entity naming
+
+We are working on improving and standardizing our entity naming. This will allow us to in the UI show entities in the right context in the future while removing some error prone magic magic mangling of entity names from the code base.
```

Comment:
```suggestion
We are working on improving and standardizing our entity naming. This will allow us to in the UI show entities in the right context in the future while removing some error-prone magic mangling of entity names from the code base.
```